src/java/fr/paris/lutece/plugins/document/web/DocumentResourceServlet.java

```diff
@@ -223,27 +222,37 @@ protected void processRequest( HttpServletRequest request, HttpServletResponse r
     /** Handles the HTTP <code>GET</code> method.
      * @param request servlet request
      * @param response servlet response
-     * @throws ServletException the servlet Exception
-     * @throws IOException the io exception
+     * @throws AppException the app exception
      */
     @Override
-    protected void doGet( HttpServletRequest request, HttpServletResponse response )
-        throws ServletException, IOException
+    protected void doGet( HttpServletRequest request, HttpServletResponse response ) throws AppException
```

Comment:
"Throws AppException" is not necessary because it's a runtime exception

Reply:
Corrected 👍